pages/common/xml-elements.md

```diff
@@ -0,0 +1,28 @@
+# xml-elements
+
+> Extract elements and structure of an XML document.
```

Comment:
Perhaps this will read better?
```suggestion
> Extract elements and their structure of an XML document.
```

Reply:
> Extract elements and display structure of an XML document.

Reply:
Resolved all places:
xml-elements.md:> Extract elements and display structure of an XML document.
xml-elements.md:- Extract elements from an XML document (producing XPATH expressions):
xml-elements.md:- Extract elements and their attributes from an XML document:
xml-elements.md:- Extract elements and their attributes and values from an XML doc: